proto/steward.proto

```diff
@@ -41,3 +42,12 @@ message ScheduleResponse {
     // The hex encoded ID of the scheduled cork
     string id = 1;
 }
+
+/*
+ * Represents a request for Steward's current status
+ */
+message StatusRequest {}
+
+message StatusResponse {
+    string version = 1;
```

Comment:
Are there other health indicators we may want to include? It doesn't necessarily mean we have to do it now, but some things to consider: last successful update times on some of the background threads, possibly some kind of error counts or flags, process start time/uptime, etc.

Reply:
I was thinking along similar lines. In the future plz.